### PR TITLE
changes according to suggestions

### PR DESCRIPTION
**Pull request by sashaboi**
Changes asked for 👍 : 
Add Proper and valid installation/usage link -- 
image

Rename the site in netlify to your component library name -- 
image

Add cursor: not allowed on disabled button -- 
image

Have more content in the horizontal card -- 
added new card : 
image

Add a proper card with badge variant. -- 

image

Add a better example of responsive image -- 
added a random grid in which the image auto fills 
image

Add code snippets for text utilities and lists
done.
Please review.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -867,4 +879,179 @@ a:visited{
...
+.before-card{
+    display: flex;
+
+}
+
+.badge-drone{
+    margin-left: -1.9rem;
+    background-color: black;
+    padding: 0.2rem;
+    color: white;
```
Comment (jatinmore) [inline style.css:1048]: instead of color name you can use variables

File: style.css [hunk 2]
```diff
@@ -867,4 +879,179 @@ a:visited{
...
+    border-radius: 0.5rem;
+    margin-bottom: 0.2rem;
+}
+.addtocart{
+    width: 200px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: linear-gradient(#40a9ff,#1890ff);
+    color: white;
```
Comment (jatinmore) [inline style.css:1028]: same goes here add variable instead of names

**Review by jatinmore — APPROVED** [review]
added some suggestions in style.css section
everything looks good

File: components.html [hunk 3]
```diff
@@ -357,24 +362,54 @@ <h3>
...
+                    style="width: 563px; height: 858px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (teja538) [inline components.html:412]: suggestion : can use the separate class for iframe for styling. so that you can reuse it.

File: components.html [hunk 4]
```diff
@@ -357,24 +362,54 @@ <h3>
...
+                    </div>
 
                 </div>
+
+                <div class="badge-drone">Pro</div>
+
             </div>
+
+
+
```
Comment (teja538) [inline components.html:404]: remove the extra spaces

**Review by teja538 — APPROVED** [review]
everything looks good. just remove the extra spaces.

File: components.html (hunk 3 above)
Comment (sashaboi) [reply components.html:412]: good call, will try doing this.

**Pull request merged**